### PR TITLE
Doc: Launch script update for acrn on qemu

### PR DESCRIPTION
**Pull request by fuzhongl**
To launch User VM, cpu_affinity parameter is needed for v2.7.

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/acrn_on_qemu.rst [hunk 1]
```diff
@@ -252,6 +252,7 @@ Bring Up User VM (L2 Guest)
       acrn-dm -A -m $mem_size -s 0:0,hostbridge \
       -s 3,virtio-blk,~/UserVM.img \
       -s 4,virtio-net,tap0 \
+      --cpu_affinity 1 \
```
Comment (gvancuts) [inline doc/tutorials/acrn_on_qemu.rst:255]: The sample script in the repo also includes the following two lines:
```
-k /home/guestl1/acrn-dm-bins/bzImage_uos \
-B "earlyprintk=serial,ttyS0,115200n8 consoleblank=0 root=/dev/vda1 rw rootwait maxcpus=1 nohpet console=tty0 console=hvc0 console=ttyS0 no_timer_check ignore_loglevel log_buf_len=16M tsc=reliable" \
```

They do not appear in this launch script snippet, does the user need to remove those?

**Review by gvancuts — CHANGES_REQUESTED** [review]
Can we also rename the scenario to `shared` (and update that in this document)?

```
$ git diff --cached 
diff --git a/misc/config_tools/data/qemu/sdc.xml b/misc/config_tools/data/qemu/shared.xml
similarity index 98%
rename from misc/config_tools/data/qemu/sdc.xml
rename to misc/config_tools/data/qemu/shared.xml
index 8f21a2d1d..3a85f8d71 100644
--- a/misc/config_tools/data/qemu/sdc.xml
+++ b/misc/config_tools/data/qemu/shared.xml
@@ -1,4 +1,4 @@
-<acrn-config board="qemu" scenario="sdc">
+<acrn-config board="qemu" scenario="shared">
   <hv>
     <DEBUG_OPTIONS>
       <RELEASE>n</RELEASE>
```

File: doc/tutorials/acrn_on_qemu.rst (hunk 1 above)
Comment (fuzhongl) [reply doc/tutorials/acrn_on_qemu.rst:255]: Yes, since OVMF will be used to launch User VM.

**Comment by fuzhongl** [comment]
Two PR about rename sdc.xml to shared.xml on qemu platform were merged:
[release_2.7 PR]: https://github.com/projectacrn/acrn-hypervisor/pull/6956
[Master PR]: https://github.com/projectacrn/acrn-hypervisor/pull/6955

File: doc/tutorials/acrn_on_qemu.rst (hunk 1 above)
Comment (gvancuts) [reply doc/tutorials/acrn_on_qemu.rst:255]: Thanks, please make this clear in the document.

File: doc/tutorials/acrn_on_qemu.rst [hunk 2]
```diff
@@ -11,8 +11,8 @@ configuration.
 
 This setup was tested with the following configuration:
 
-- ACRN hypervisor: ``v2.6`` tag
-- ACRN kernel: ``v2.6`` tag
+- ACRN hypervisor: ``v2.7`` tag
+- ACRN kernel: ``v2.7`` tag
 - QEMU emulator version: 4.2.1
 - Service VM/User VM OS: Ubuntu 20.04
```
Comment (gvancuts) [inline doc/tutorials/acrn_on_qemu.rst:17]: This is not what the instructions below use, we seem to be using Ubuntu 18.04 instead:
```
--os-variant ubuntu18.04 \
...
--location 'http://archive.ubuntu.com/ubuntu/dists/bionic/main/installer-amd64/' \
```

**Review by gvancuts — CHANGES_REQUESTED** [review]

File: doc/tutorials/acrn_on_qemu.rst (hunk 2 above)
Comment (fuzhongl) [reply doc/tutorials/acrn_on_qemu.rst:17]: @gvancuts 
Ubuntu 20.04 is the native OS; not Service VM/User VM OS
acrn on qemu is verified for Service VM OS: 18.04

Any comments about following change?
From:
`Service VM/User VM OS: Ubuntu 20.04`
To:
`Ubuntu 20.04`

Thanks!

Comment (gvancuts) [reply doc/tutorials/acrn_on_qemu.rst:17]: My preference would be if we could consolidate onto a single version of Ubuntu, i.e. 20.04. Second best thing would be to keep these different version but specify them both, i.e.:
```
Host OS: Ubuntu 20.04
Service VM / User VM: Ubuntu 18.04
````

**Review by NanlinXie — APPROVED** [review]

**Comment by NanlinXie** [comment]
@gvancuts , any more comments?

**Comment by gvancuts** [comment]
> @gvancuts , any more comments?

There is still an open ask from me to make it clear in this doc that a couple of lines from the sample script need to be removed by the user - see https://github.com/projectacrn/acrn-hypervisor/pull/6953#discussion_r764663158. It should be good to go after that.

**Comment by dbkinder** [comment]
@NanlinXie @wenlingz CI still has this tagged as "no Signed-off-by"

**Comment by fuzhongl** [comment]
> > @gvancuts , any more comments?
> 
> There is still an open ask from me to make it clear in this doc that a couple of lines from the sample script need to be removed by the user - see [#6953 (comment)](https://github.com/projectacrn/acrn-hypervisor/pull/6953#discussion_r764663158). It should be good to go after that.

@gvancuts Your concern is covered by following PR which uses ovmf to launch User VM
Doc: launch L2 guest with ovmf for ACRN on QEMU #6576

**Comment by dbkinder** [comment]
> > > @gvancuts , any more comments?
> > 
> > 
> > There is still an open ask from me to make it clear in this doc that a couple of lines from the sample script need to be removed by the user - see [#6953 (comment)](https://github.com/projectacrn/acrn-hypervisor/pull/6953#discussion_r764663158). It should be good to go after that.
> 
> @gvancuts Your concern is covered by following PR which uses ovmf to launch User VM Doc: launch L2 guest with ovmf for ACRN on QEMU #6576

I think @gvancuts is looking for something in this document that says you need to edit and remove a couple of lines in the sample script.  If this is indeed needed, I agree we should make that explicit here.

**Comment by fuzhongl** [comment]
> > > > @gvancuts , any more comments?
> > > 
> > > 
> > > There is still an open ask from me to make it clear in this doc that a couple of lines from the sample script need to be removed by the user - see [#6953 (comment)](https://github.com/projectacrn/acrn-hypervisor/pull/6953#discussion_r764663158). It should be good to go after that.
> > 
> > 
> > @gvancuts Your concern is covered by following PR which uses ovmf to launch User VM Doc: launch L2 guest with ovmf for ACRN on QEMU #6576
> 
> I think @gvancuts is looking for something in this document that says you need to edit and remove a couple of lines in the sample script. If this is indeed needed, I agree we should make that explicit here.

@dbkinder 
Thanks for your comments; the whole launch script is pasted in the doc now. Don't refer to the linked script any more since it will be removed from next release.

**Review by dbkinder — APPROVED** [review]
LGTM

**Pull request merged**